projects/demo-integrations/cypress/tests/kit/stepper/stepper.spec.ts

```diff
@@ -0,0 +1,37 @@
+describe('Stepper', () => {
+    describe(
+        'scroll internal scrollable area to active/focused step',
+        {scrollBehavior: false},
+        () => {
+            it('horizontal', () => {
+                cy.viewport('iphone-x');
+                cy.tuiVisit('/navigation/stepper/API?orientation=horizontal');
```

Comment:
Can we reuse from here https://github.com/Tinkoff/taiga-ui/pull/1811/files?

Reply:
it is just "any small screen size"
It can be even small desktop)